packages/virtualized-lists/Lists/__tests__/__snapshots__/VirtualizedList-test.js.snap

```diff
@@ -3122,12 +3122,53 @@ exports[`gracefully handles negative initialScrollIndex 1`] = `
       />
     </View>
     <View
-      style={
-        Object {
-          "height": 60,
-        }
-      }
-    />
+      onFocusCapture={[Function]}
```

Comment:
Is this change expected? It looks this is the snapshot from https://github.com/facebook/react-native/blob/a27270f492400c545db4a2fd15d62acfc2051c89/packages/virtualized-lists/Lists/__tests__/VirtualizedList-test.js#L828

which lists `initialNumToRender` as 4

Reply:
It looks like the snapshot is taken after we run fake layout and timers (after initial render), so I think this is showing that we are now rendering cells we previously failed to.